webview/src/webview_ios.mm

```diff
@@ -70,15 +71,18 @@ void Clear() {
 
 @implementation WebViewDelegate
 
+bool m_AllowSiteAction = false;
```

Comment:
@britzl it was defined here, but well, I think I get what you guys mean here, lemme clean up...